src/ppm.cpp

```diff
@@ -775,7 +785,7 @@ int PPM::operator()(void) {
     } catch ( exception& e ) {
 
         // don't use logger in case we cannot configure it correctly.
-        cerr << "Fatal Exception: " << e.what() << '\n';
+        ilogger->error("Fatal Exception: {}", e.what());
```

Comment:
This one may need to be left with the cerr from the comment on 787.

Reply:
This has been reverted.